### ISSUE TITLE
I can't update.

### ISSUE THREAD

**Issue by AlexanderTheCrafter** [TheSharks/WildBeast#326]
I cant update from 4.5.0 to 6.0.0. How do I update?
Also... what happened to 5.0.0?

**Comment by AlexanderTheCrafter** [comment]
![image](https://user-images.githubusercontent.com/40205201/41292512-cd0888b0-6e20-11e8-92f4-a116243a66ed.png)
Also... Does this mean that I can modify WildBeast and not have to make it open source on GitHub?

**Comment by linuswillner** [comment]
You cannot directly update from 4.5.0 to 6.0.0, the code is not compatible in any way. For further details, see https://docs.thesharks.xyz/migrating/.

Version 5 was in development internally, but development proceeded to what would become version 6 before it was ever released to the public. This explains the version jump.

**Comment by Dougley** [comment]
To comment on the license question, you must disclose your source code when you distribute, publish or serve modified or derivative versions of WildBeast.

**Comment by linuswillner** [comment]
Adding on to what @Dougley said here, the relevant sections of the license TL;DR read as follows:

Permissions of this strongest copyleft license are conditioned on **making available complete source code of licensed works and modifications**, which include larger works using a licensed work, under the same license. **Copyright and license notices must be preserved**. Contributors provide an express grant of patent rights. **When a modified version is used to provide a service over a network, the complete source code of the modified version must be made available.**

In other words: Yes, you may modify WildBeast and use it privately. However:

- You must preserve the license (And not change it) and copyright notice
- You must make the source code of your modified version available to the public, and your end users have a right to see it (Or request to do so) at any time

The easiest way to comply with these requirements is by forking this repository and pushing your own changes to it.

**Comment by AlexanderTheCrafter** [comment]
Ok... Thanks @linuswillner @Dougley
